common/protob/messages-neo.proto

```
@@ -0,0 +1,63 @@
+syntax = "proto2";
+package hw.trezor.messages.neo;
```

Buf (1.47.2)

Files with package "hw.trezor.messages.neo" must be within a directory "hw/trezor/messages/neo" relative to root but were in directory "common/protob".

(PACKAGE_DIRECTORY_MATCH)